### PR TITLE
Configurable context menu

### PR DESCRIPTION
**Pull request by jyothishnt**

### CODE REVIEW

**Comment by jyothishnt** [comment]
New feature + Fix #57

File: src/scripts/PhyloCanvas.js [hunk 1]
```diff
@@ -343,37 +347,55 @@ var PhyloCanvas = (function () {
     this.div.style.position = 'fixed';
     this.div.style.border = '1px solid #CCCCCC';
     this.div.style.background = '#FFFFFF';
+    // this.div.style.fontSize = 'px';
+    this.div.style.letterSpacing = '0.5px';
```
Comment (richardgoater) [inline src/scripts/PhyloCanvas.js]: not sure about specifying this in pixels, is it even used?

File: src/scripts/PhyloCanvas.js [hunk 2]
```diff
@@ -343,37 +347,55 @@ var PhyloCanvas = (function () {
...
-      internal: true,
-      leaf: false
-    }, {
-      text: 'Rotate Branch',
-      handler: 'rotate',
-      internal: true,
-      leaf: false
-    }];
+    this.elements = [];
+    if (options.length > 0) {
```
Comment (richardgoater) [inline src/scripts/PhyloCanvas.js]: Think you might need to check for the existence of options here, something like `if (options && options.length > 0)`

**Comment by richardgoater** [comment]
@jyothishnt Would be good to add a demo for this but I guess that will go on the docs site?

**Comment by jyothishnt** [comment]
Done!
Already created a demo for this in the docs site. :) Thanks Rich

**Pull request merged**